app/models/review.rb

```diff
@@ -0,0 +1,11 @@
+class Review < ActiveRecord::Base
```

Comment:
I have no idea what this means. If you could do me a favor and find the rule for this, explain it, and then please remove it in a separate PR. :D